src/Microsoft.Azure.Functions.PowerShellWorker.csproj

```diff
@@ -1,4 +1,4 @@
-<!--
+﻿<!--
```

Comment:
nipickt: did anything change in this file? Seems like nothing changed. If so - can we instruct git to "revert this file" so that it doesn't register a change in the PR?